snap/snapcraft.yaml

```diff
@@ -99,6 +100,7 @@ parts:
       - kodi-wayland
       - kodi-visualization-spectrum
       - kodi-repository-kodi
+      - kodi-inputstream-adaptive
```

Comment:
Which repos contain this? It doesn't seem to be in the 20.04 archive - which we use for arm64/armhf.

Reply:
Right… I've `ppa:team-xbmc` enabled… but it's amd64 only. I suppose we could have this "try: ['kodi-inputstream-adaptive']" and add the PPA?

Reply:
We already add the PPA (for Intel), I suggest:
```
    - on amd64: kodi-inputstream-adaptive
    - on i386: kodi-inputstream-adaptive
```
With, maybe, some comments.

Reply:
OK with this?

```suggestion
      - try: [kodi-inputstream-adaptive] # only available for x86
```